main.go

```diff
@@ -103,7 +103,7 @@ func main() {
 		Scheme:       mgr.GetScheme(),
 		PlatformInfo: platformInfo,
 		Namespace:    watchNamespace,
-	}).SetupWithManager(mgr, bgpType); err != nil {
+	}).SetupWithManager(mgr, bgpType, os.Getenv("ENABLE_WEBHOOK")); err != nil {
```

Comment:
Not blocking, but I'd prefer to have the parameter as a bool (or even better, an enumerate) rather than passing in the string and parsing it inside.